.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,85 @@
+name: "Benchmark on Comment"
+
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  Benchmark:
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.8]
+        os: [self-hosted]
+
+    name: Benchmark
+    # Only run if it#s a PR and the comment contains /Benchmark
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/Benchmark') && contains('["vwxyzjn", "younesbelkada", "lvwerra"]', github.actor)
```

Comment:
Note there is a restriction that only `vwxyzjn", "younesbelkada", "lvwerra"` can trigger CI

Reply:
Sounds great ! We can always add more maintainers later if needed